### PR TITLE
dcp-669 added new archiveResult property to File class

### PR DESCRIPTION
**Pull request by prabh-t**

### CODE REVIEW

File: src/main/java/org/humancellatlas/ingest/file/FileArchiveResult.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import java.time.Instant;
+import lombok.Data;
+
+@Data
+public class FileArchiveResult {
+    private Instant lastArchived;
+    private Boolean compressed;
+    private String md5;
+    private String enaUploadPath;
+    private Boolean success;
```
Comment (aaclan-ebi) [inline src/main/java/org/humancellatlas/ingest/file/FileArchiveResult.java:12]: I think this is redundant with the error attribute. It's probably better to just retain the error attribute and remove this. If there's no error, that means archiving is successful.

File: src/main/java/org/humancellatlas/ingest/file/FileArchiveResult.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package org.humancellatlas.ingest.file;
+
+import java.time.Instant;
+import lombok.Data;
+
+@Data
+public class FileArchiveResult {
+    private Instant lastArchived;
+    private Boolean compressed;
```
Comment (aaclan-ebi) [inline src/main/java/org/humancellatlas/ingest/file/FileArchiveResult.java:9]: what does compressed mean?

Comment (aaclan-ebi) [reply src/main/java/org/humancellatlas/ingest/file/FileArchiveResult.java:9]: why is it needed?

Comment (prabh-t) [reply src/main/java/org/humancellatlas/ingest/file/FileArchiveResult.java:9]: it indicates the file was compressed during archiving.

Comment (aaclan-ebi) [reply src/main/java/org/humancellatlas/ingest/file/FileArchiveResult.java:9]: is that info going to be passed to ENA?

**Review by aaclan-ebi — APPROVED** [review]
LGTM, Please also add ticket info

**Review by jacobwindsor — APPROVED** [review]

**Pull request merged**